README.md

```diff
@@ -69,6 +69,7 @@ The benefits of `exfmt`:
 ## Contents
 
 - [Usage](#usage)
+- [Editor Integration](#editorintegration)
```

Comment:
I think you need a dash in the anchor (`editor-integration`) to work with the heading below.
You can check it working in the `Files changed` tab and hitting the `View` button on the top right! ;-)